app/views/medical_records/_form.html.erb

```diff
@@ -31,7 +31,19 @@
     <%= form.text_field :current_state %>
   </div>
 
+  <div>
+  <%= form.label :notes, style: "display: block" %>
+    <%= form.text_area :notes, rows: 5 %>
+  </div>
+
   <div>
     <%= form.submit %>
   </div>
+  <% if @medical_record.image.attached? %>
+  <%= image_tag("Odontogram.jpg", alt: "odontograma", class:"d-block w-100") %>
```

Comment:
If you want to display the recorded image, you need to use 
```suggestion
  <%= image_tag(@medical_record.image, alt: "odontograma", class:"d-block w-100") %>
```
In order to not display always the same image.
But I'm not sure understand what you want to do.